src/components/templates/AnyWayGridFilterMenu.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { FunctionComponent } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import CalendarTodayIcon from '@material-ui/icons/CalendarToday';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+import { Button, Tooltip } from '@material-ui/core';
+interface IProps {
+    
+}
+
+const useStyles = makeStyles(theme => ({
+    formControl: {
+      margin: theme.spacing(1),
+      minWidth: 120,
+    },
+    selectEmpty: {
+      marginTop: theme.spacing(2),
+    },
+  }));
+const AnyWayGridFilterMenu: FunctionComponent<IProps> = ({ children }) => {
+    const classes = useStyles();
+    const [timeBack, setTimeBack] = React.useState(3);
+    const [timeRangeActive,setTimeRangeActive] = React.useState(false)
+    const [selectedDate, setSelectedDate] = React.useState(new Date());
+    const [selectedDateEnd, setSelectedDateEnd] = React.useState(new Date());
+
+    function  handleDateChange (d : any) {
+      setSelectedDate(d);
+      console.log('chosen date', d)
+    };
+    function  handleDateChangeEnd (d : any) {
+      setSelectedDateEnd(d);
+    };
+    const devStyles = {
+        width: '100%',
+        height : '50px',
+        display: 'inline-block'
+        
+    }
+    const handleChange = (event:any) : void => {
+        setTimeBack(event.target.value);
+        
+      };
+
+    const handleTimeRangeClick = (event:any) : void => {
+      setTimeRangeActive(!timeRangeActive);
+      
+    };
+    return  <div style={devStyles} >
```

Comment:
Please add parentheses for multiple lines of JSX.
Also, please fix the indentation of what you return.